### DISCUSSION TITLE
Multiscale FEM Modeling

### DISCUSSION THREAD

**Discussion by manikumd22** [idaholab/moose#26332] [Q&A Modules: Solid mechanics]
I am working on multiscale modeling (computational homogenization scheme), and in multiscale modeling, we apply the PBC to an RVE using a deformation gradient tensor, and that RVE (sub ap) will give first Piola Kirchoff stress using the homogenization technique over RVE for each of the macro gauss points 

In my case, I have a CSV file that will take the deformation gradient tensor as input and it will give the first Piola stress tensor as output that will be used at each gauss point. I'm struggling with this

Can you please help me with this?

**Comment by GiudGiud** [comment]
@recuero @dschwen on RVE
